kubectl-plugin/README.md

```diff
@@ -2,56 +2,25 @@
 
 Kubectl plugin/extension for Kuberay CLI that provides the ability to manage ray resources.
 
-## Installation
-
-<!-- 1. Check [release page](https://github.com/ray-project/kuberay/releases) and download the necessary binaries. -->
-1. Run `go build cmd/kubectl-ray.go`
-2. Move the binary, which will be named `kubectl-ray` to your `PATH`
-
 ## Prerequisites
 
 1. Make sure there is a Kubernetes cluster running with KubeRay installed.
 2. Make sure `kubectl` has the right context.
 
-## Usage
-
-### Retrieve Ray Clusters
-
-    Usage:
-    ray cluster get [flags]
-
-    Aliases:
-    get, list
-
-    Description:
-    Retrieves the ray cluster information.
+## Installation
 
-    ```
-    $ kubectl ray cluster get
-    NAME                       NAMESPACE   DESIRED WORKERS   AVAILABLE WORKERS   CPUS   GPUS   TPUS   MEMORY   AGE
-    random-kuberay-cluster   default     1                 1                   5      1      0      24Gi     13d
-    ```
+### Compiling from source
 
-### Retrieve Ray Cluster Head Logs
+1. Run `go build cmd/kubectl-ray.go`
+2. Move the binary, which will be named `kubectl-ray` to your `PATH`
 
-    Usage:
-    log (RAY_CLUSTER_NAME) [--out-dir directory] [--node-type all|head|worker]
-    **Currently only `head` is supported**
+### Using Krew
 
-    Aliases:
-    log, logs
+1. Install [Krew](https://krew.sigs.k8s.io/docs/user-guide/setup/install/).
+2. (TODO: Replace this step with the installation command).
```

Comment:
To clarify, this TODO will be blocked until we release KubeRay v1.3 which includes the binray release of the plugin right?

Reply:
I mentioned a discussion above here

https://github.com/ray-project/kuberay/pull/2384#discussion_r1761237786

I don't know whether we want to use the same version for `ray-operator` and `kubectl-plugin` or not. If we want to use different versioning for them, then this will block untill to `kubectl-plugin` v0.1.0 release. If we want to use the same version, then yes this will block until the release of KubeRay v1.3.0.

cc @kevin85421

Reply:
Personally I think we should keep the releases together but I'm also fine to do it separately. Will defer to you and @chiayi

Reply:
> Personally I think we should keep the releases together but I'm also fine to do it separately.

I also prefer to keep the releases together to simplify the release process, unless there is a strong benefit to separating them.